src/Avalonia.Base/Rendering/Composition/Expressions/ExpressionVariant.cs

```diff
@@ -21,7 +21,9 @@ internal enum VariantType
         Matrix3x2,
         Matrix4x4,
         Quaternion,
-        Color
+        Color,
+        RelativePoint,
```

Comment:
Should probably add ctors to FFI too, e. g. https://github.com/AvaloniaUI/Avalonia/blob/51987447d3e8571cd453efaef94c4fe91383968c/src/Avalonia.Base/Rendering/Composition/Expressions/BuiltInExpressionFfi.cs#L131-L136

Not sure how to handle RelativeUnit there, might need a separate VariantType and separate ffi functions as enum value "ctors".